c++/src/hello_world.cpp

```diff
@@ -9,4 +9,6 @@
 
 #include <iostream>
 
-int main(int argc, char **argv) { std::cout << "Hello World!!" << std::endl; }
\ No newline at end of file
+int main(int argc, char **argv) {
```

Comment:
I really like this `char**` syntax because of stuff...! @IainBrookshaw This is ok!